include/ml_dsa/internals/poly/sampling.hpp

```diff
@@ -61,20 +61,20 @@ expand_a(std::span<const uint8_t, 32> rho, std::span<ml_dsa_field::zq_t, k * l *
   }
 }
 
-// Uniform rejection sampling k -many degree-255 polynomials s.t. each coefficient of those polynomials ∈ [-η, η].
+// Uniform rejection sampling k -many degree-255 polynomials s.t. each coefficient of those polynomials ∈ [-eta, eta].
 //
 // Sampling is performed deterministically, by seeding Shake256 Xof with 64 -bytes seed and two nonce bytes, whose
 // starting value is provided ( see template parameter ). Consecutive nonces are computed by adding 1 to previous value.
 //
 // Note, sampled polynomial coefficients are kept in canonical form.
 //
 // See algorithm 33 of ML-DSA standard @ https://doi.org/10.6028/NIST.FIPS.204.
-template<uint32_t η, size_t k, uint16_t nonce>
+template<uint32_t eta, size_t k, uint16_t nonce>
 static inline constexpr void
 expand_s(std::span<const uint8_t, 64> rho_prime, std::span<ml_dsa_field::zq_t, k * ml_dsa_ntt::N> vec)
-  requires(ml_dsa_params::check_η(η) && ml_dsa_params::check_nonce(nonce))
+  requires(ml_dsa_params::check_eta(eta) && ml_dsa_params::check_nonce(nonce))
 {
-  constexpr auto eta = ml_dsa_field::zq_t(η);
+  constexpr auto eta = ml_dsa_field::zq_t(eta);
```

Comment:
This won't work, see the test run failure on CI.

Reply:
Should be good now